app/src/main/java/com/codesoom/assignment/App.java

```diff
@@ -1,10 +1,31 @@
 package com.codesoom.assignment;
 
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
 public class App {
+    private final static int LOCAL_HOST = 8000;
```

Comment:
포트에 대해 공부할 좋은 기회에요. 조금 창피하실 수도 있어요. 그치만 괜찮아요. 이번 기회에 확실히 알고 지나가면 되는 거죠!
```suggestion
    private final static int PORT = 8000;
```

Reply:
아;; 포트군요... 넵 맞습니다ㅎㅎ 감사합니다 이번 기회에 8000번이 PORT라는 것을 알게 됬네요^^

Reply:
시간나실 때 포트를 명시한 공식 문서를 찾아서 읽어보세요. 그게 어려우면 제가 작성한 문서라도 한번 읽어보세요 ㅎㅎ
https://johngrib.github.io/wiki/why-http-80-https-443/

Reply:
감사합니다 꼭 읽어보도록 하겠습니다 ^^